advanced-navigation/app/src/main/java/com/here/android/example/advanced/navigation/MapFragmentView.java

```diff
@@ -401,6 +408,15 @@ public void onRouteUpdated(Route route) {
                 }
             };
 
+    final private NavigationManager.LaneInformationListener
+            m_laneInformationListener = new NavigationManager.LaneInformationListener() {
+        @Override
+        public void onLaneInformation(@NonNull List<LaneInformation> items,
+                                      @Nullable RoadElement roadElement) {
+            LaneInfoUtils.displayLaneInformation(m_laneInfoView, items);
+        }
+    };
+
     void onDestroy() {
```

Comment:
add removal of the m_laneInformationListener -> navigationManager.removeLaneInformationListener()